apps/docs/components/JwtGenerator/JwtGenerator.tsx

```diff
@@ -70,7 +70,7 @@ export default function JwtGenerator({}) {
     else setToken(anonToken)
   }
   const generate = () => {
-    const signedJWT = KJUR.jws.JWS.sign(null, JWT_HEADER, token, jwtSecret)
+    const signedJWT = KJUR.jws.JWS.sign("HS256", JWT_HEADER, token, jwtSecret)
```

Comment:
Any reason to prefer being explicit about the algorithm here? If left to null, it will use the algorithm from the header, which I just tested and produces exactly the same effect, but is less likely to drift out of sync.

Reply:
Hey @charislam 
Thanks so much for the reviews, With this part of the code I was just to make the change based on this article on the repository for the jsrsasign package. https://github.com/kjur/jsrsasign/wiki/Tutorial-for-JWT-generation.

Also I spent around 3 hour trying to debug the JwtGenerator locally because no matter what I did it always produced different results than [jwt.io](https://jwt.io) and from my testing the only secret keys that have every worked for me and others have been those generated on jwt.io
https://github.com/orgs/supabase/discussions/19603